### ISSUE TITLE
Pre-trained word embeddings example - the datasets include headers which contain the labels

### ISSUE THREAD

**Issue by vijayzemoso** [fchollet/keras#5065]
First of all thanks for the wonderful example to get started with Embeddings. 
While the goal of the example is to introduce to the world of Embeddings - I happened to notice that every single dataset includes the headers in the email - most of which should be fine except for the "Newsgroups" which is a dead giveaway on the output label 
 
Perhaps the blog entry can be modified to reflect this fact - so that readers can take the accuracy of the output as not being representative and maybe the example can be modified to ignore the headers -till the one saying Lines:

The suggestion is just to prevent mismatch of expectations w.r.t very high accuracy with training on just 2 epochs

Example eader in a sample dataset
Path: cantaloupe.srv.cs.cmu.edu!das-news.harvard.edu!ogicse!network.ucsd.edu!news.service.uci.edu!ucivax!megatek!randy
From: randy@megatek.com (Randy Davis)
Newsgroups: rec.motorcycles
Subject: Re: A Miracle in California
Message-ID: <1993Apr5.223941.11539@megatek.com>
Date: 5 Apr 93 22:39:41 GMT
Article-I.D.: megatek.1993Apr5.223941.11539
References: <95.602CV3dTx01@JUTS.ccc.amdahl.com> <1ppvof$92a@seven-up.East.Sun.COM>
Sender: randy@megatek.com (Randy Davis)
Reply-To: randy@megatek.com
Organization: Megatek Corporation, San Diego, California
Lines: 15

I am not too familiar with the history of this dataset and objective so please pardon my ignorance if the inclusion of the header is intentional)

- [ X] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [ X] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [X ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ X] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).
